analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Constraints/SerializationConstraint.cs

```diff
@@ -18,18 +18,15 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using SonarAnalyzer.SymbolicExecution.Constraints;
+namespace SonarAnalyzer.SymbolicExecution.Constraints;
 
-namespace SonarAnalyzer.SymbolicExecution.Sonar.Constraints
+internal sealed class SerializationConstraint : SymbolicConstraint
 {
-    internal sealed class SerializationConstraint : SymbolicConstraint
-    {
-        public static readonly SerializationConstraint Unsafe = new(ConstraintKind.SerializationUnsafe);
-        public static readonly SerializationConstraint Safe = new(ConstraintKind.SerializationSafe);
+    public static readonly SerializationConstraint Unsafe = new(ConstraintKind.SerializationUnsafe);
+    public static readonly SerializationConstraint Safe = new(ConstraintKind.SerializationSafe);
 
-        public override SymbolicConstraint Opposite =>
-            this == Safe ? Unsafe : Safe;
+    public override SymbolicConstraint Opposite =>
+        this == Safe ? Unsafe : Safe;
```

Comment:
No need for an `Opposite` implementation. It can just return `null`.

Reply:
This constraint is shared with the old rule implementation, which already had the Opposite implementation. I would rather not touch it.